client/cmd/utils/flags.go

```diff
@@ -848,8 +845,6 @@ func SetNodeConfig(ctx *cli.Context, cfg *node.Config, kowalaCfg *knode.Config)
 	switch {
 	case ctx.GlobalIsSet(DataDirFlag.Name):
 		cfg.DataDir = ctx.GlobalString(DataDirFlag.Name)
-	case ctx.GlobalBool(DevModeFlag.Name):
```

Comment:
do we risk overriding the current data?

Reply:
Yes, it will use the same storage. That the same behaviour as the --testnet flag, which is why I removed those lines.